### PR TITLE
Add draft server implementation using `Spark java`

### PR DESCRIPTION
**Pull request by kuzmavictor**
This PR contains a simple demonstration of `GET` request.

### CODE REVIEW

**Comment by kuzmavictor** [comment]
@denysLavrienov @AndriiSameliuk @BohdanDudkaTD PTAL.

File: server/src/main/java/io/javaclasses/filestorage/server/HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package io.javaclasses.filestorage.server;
+import static spark.Spark.*;
```
Comment (BohdanDudkaTD) [inline server/src/main/java/io/javaclasses/filestorage/server/HelloWorld.java]: Please, import a single `get` method instead of the whole package.

File: server/src/main/java/io/javaclasses/filestorage/server/HelloWorld.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package io.javaclasses.filestorage.server;
+import static spark.Spark.get;
```
Comment (armiol) [inline server/src/main/java/io/javaclasses/filestorage/server/HelloWorld.java]: Please use IDEA's autoformatting.

File: server/src/main/java/io/javaclasses/filestorage/server/HelloWorld.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+/**
+ * This class demonstrates a get request.
```
Comment (armiol) [inline server/src/main/java/io/javaclasses/filestorage/server/HelloWorld.java]: These requests are `GET` requests. Please adjust the PR description accordingly.

File: web-client/app/component/popper-component.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import {Component} from './component.js';
-import Popper from '../../../node_modules/popper.js/dist/esm/popper.js';
+import Popper from '../../node_modules/popper.js/dist/esm/popper.js';
```
Comment (armiol) [inline web-client/app/component/popper-component.js]: Please revert. This change does not belong to this PR.

File: server/src/main/java/io/javaclasses/filestorage/server/HelloWorld.java [hunk 5]
```diff
@@ -0,0 +1,11 @@
+package io.javaclasses.filestorage.server;
```
Comment (armiol) [inline server/src/main/java/io/javaclasses/filestorage/server/HelloWorld.java]: Please describe the package.

File: server/src/main/java/io/javaclasses/filehub/HelloWorld.java [hunk 6]
```diff
@@ -0,0 +1,19 @@
+package io.javaclasses.filehub;
+
+import static spark.Spark.get;
+
+/**
+ * This class demonstrates a {@code GET} request.
```
Comment (armiol) [inline server/src/main/java/io/javaclasses/filehub/HelloWorld.java]:
```
This class defines an endpoint handling {@code GET} requests.
```
Feel the difference.

File: server/src/main/java/io/javaclasses/filehub/package-info.java [hunk 7]
```diff
@@ -0,0 +1,5 @@
+/**
+ * This package provides environment for server part  of the FileHub system.
+ */
+
+package io.javaclasses.filehub;
```
Comment (armiol) [inline server/src/main/java/io/javaclasses/filehub/package-info.java]: Please fix EOF.

File: server/src/main/java/io/javaclasses/filehub/HelloWorld.java [hunk 8]
```diff
@@ -0,0 +1,19 @@
+package io.javaclasses.filehub;
+
+import static spark.Spark.get;
+
+/**
+ * This class defines an endpoint handling {@code GET} requests.
+ */
+public class HelloWorld {
+    /**
```
Comment (armiol) [inline server/src/main/java/io/javaclasses/filehub/HelloWorld.java]: Please have an empty line above.
